docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -74,7 +74,7 @@ It is also possible to configure all of the Kotlin compilation tasks in the proj
 
 ```kotlin
 tasks.withType<org.jetbrains.kotlin.gradle.dsl.KotlinCompile>().configureEach {
```

Comment:
If here (and in other places) is general example - better to use `org.jetbrains.kotlin.gradle.tasks.KotlinCompilationTask` type. If it is about JVM platform - `org.jetbrains.kotlin.gradle.tasks.KotlinJvmTask`

Reply:
Ok, I replaced it; please check in the upcoming commit.

Reply:
Does `import org.jetbrains.kotlin.gradle.tasks.KotlinCompile` stay the same?